docker-compose-dev.yml

```diff
@@ -0,0 +1,20 @@
+version: '2'
+services:
+  shogun-boot:
+    build:
+      context: ../shogun/shogun-boot/
```

Comment:
As this may vary between local environments we should add a section to the README regarding the suggested/expected directory structure, something like: 

## Recommended directory structure

```bash
shogun-directory/
├── shogun (https://github.com/terrestris/shogun)
├── shogun-docker (this repository)
└── …
```

Reply:
See #15.